### PR TITLE
ci: sdk-codegen ci

### PR DESCRIPTION
**Pull request by joeldodge79**

### CODE REVIEW

File: .github/workflows/codegen-ci.yml [hunk 1]
```diff
@@ -0,0 +1,146 @@
+name: Typescript SDK CI
```
Comment (jkaster) [inline .github/workflows/codegen-ci.yml:1]:
```suggestion
name: Codegen CI
```

File: .github/workflows/codegen-ci.yml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      - name: Install Node ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: |
+          yarn
+          yarn lint:es --quiet
+          yarn build
```
Comment (jkaster) [inline .github/workflows/codegen-ci.yml:78]:
```suggestion
          yarn build
          yarn lint:es --quiet
```

build should be attempted first to ensure import references for linting are successful.

File: .github/workflows/codegen-ci.yml [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+          echo "client_secret=secret" >> looker.ini
+          echo "verify_ssl=False" >> looker.ini
+          echo "timeout=31" >> looker.ini
+          echo "[Looker31]" >> looker.ini
+          echo "base_url=https://self-signed.looker.com:19999" >> looker.ini
+          echo "verify_ssl=False" >> looker.ini
+          echo "timeout=30" >> looker.ini
+
+      - name: Run unit tests
+        run: yarn jest "packages/sdk-codegen(-utils|-scripts)/src" --reporters=default --reporters=jest-junit
```
Comment (jkaster) [inline .github/workflows/codegen-ci.yml:94]: Do you think we need `/src` here? We may have tests under a `/test` folder in the future, perhaps. We could always change it then

**Review by jkaster — CHANGES_REQUESTED** [review]
Minor tweaks requested

File: .github/workflows/codegen-ci.yml (hunk 3 above)
Comment (joeldodge79) [reply .github/workflows/codegen-ci.yml:94]: yeah, I was just copy/paste from tssdk-ci

**Review by jkaster — APPROVED** [review]
LGTM

**Pull request merged**